artifactory/commands/generic/download.go

```diff
@@ -111,10 +112,18 @@ func (dc *DownloadCommand) Run() error {
 		dc.result.SetFailCount(0)
 		return err
 	} else if dc.SyncDeletesPath() != "" {
-		walkFn := createSyncDeletesWalkFunction(filesInfo)
-		err = fileutils.Walk(dc.SyncDeletesPath(), walkFn, false)
+		absSyncDeletesPath, err := filepath.Abs(dc.SyncDeletesPath())
 		if err != nil {
-			return err
+			return nil
```

Comment:
Any reason why return nul here?
We should also have CheckError on the returned err.